src/parsing/transaction.ts

```diff
@@ -215,14 +226,74 @@ export function parseSigningMode(mode: TransactionSigningMode): TransactionSigni
 export function parseSignTransactionRequest(request: SignTransactionRequest): ParsedSigningRequest {
     const tx = parseTransaction(request.tx)
     const signingMode = parseSigningMode(request.signingMode)
+    const additionalWitnessPaths = request.additionalWitnessPaths ? request.additionalWitnessPaths.map(path => parseBIP32Path(path, InvalidDataReason.INVALID_PATH)) : []
```

Comment:
I think this could be written also like this:
```suggestion
    const additionalWitnessPaths = request.additionalWitnessPaths?.map(path => parseBIP32Path(path, InvalidDataReason.INVALID_PATH)) || []
```